pkgs/by-name/pl/playdate-sdk/package.nix

```diff
@@ -0,0 +1,91 @@
+{
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchurl,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  pkgs,
+  stdenv,
+}:
+
+let
+  version = "2.5.0";
+in
+
+stdenv.mkDerivation rec {
+  pname = "playdate-sdk";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://download.panic.com/playdate_sdk/Linux/PlaydateSDK-${version}.tar.gz";
+    sha256 = "d5bee3ee590dd7a60ee04516c993d9f8f3c2f526b7005a0de5cf3802b4c65e89";
```

Comment:
At present, it is *not recommended to use sha256 as a parameter for fetcher,* but to **use the hash of SRI mode as a parameter**.

you could use `nix hash to-sri --type sha256 $(nix-prefetch-url ''$1)` to generate SRI hash

Reply:
```suggestion
    hash = "sha256- ... =";
```